### ISSUE TITLE
add `virtual' to interface destructor for generated service/proxy adaptors

### ISSUE THREAD

**Issue by AndreyBarmaley** [Kistler-Group/sdbus-cpp#202]
In an inherited class without a destructor, I get sigabrt:

```
 class MyProxy : public sdbus::ProxyInterface{}

std::unique_ptr<MyProxy> ptr; // <-- SIGABRT
```

**Comment by sangelovic** [comment]
@AndreyBarmaley The code is not compilable. There is no `sdbus::ProxyInterface` class in sdbus-c++. Do you mean `sdbus::ProxyInterfaces` template class? In that case angle brackets with the list of proxy interfaces is missng.  Can you please be more specific and provide more description and full minimal example causing the abort?

**Comment by sangelovic** [comment]
If the problem is addressed in #203 , then I'll close this issue, because #203 is not a solution...
